source/compliance/hipaa/malware-detection.rst

```diff
@@ -9,14 +9,13 @@ Malware detection
 Wazuh offers several capabilities that support malware detection. The following methods achieve these detections:
 
 - :doc:`Finding patterns in the endpoint that do not match expected behavior </user-manual/capabilities/malware-detection/index>`
-- :doc:`Integrating with VirusTotal to detect and remove malware </proof-of-concept-guide/detect-remove-malware-virustotal>`
 - :doc:`Integrating with YARA to detect malware </proof-of-concept-guide/detect-malware-yara-integration>`
 - `Using constant database (CDB) lists to detect and remove malicious files <https://wazuh.com/blog/detecting-and-responding-to-malicious-files-using-cdb-lists-and-active-response/>`__
 
 These components of Wazuh help to comply with the following HIPAA sections:
 
 - **Security Awareness and Training §164.308(a)(5)(i) - Protection from Malicious Software**: *“Procedures for guarding against, detecting, and reporting malicious software.”*
   
-  This section of the HIPAA standard requires you to have procedures to detect and remove malicious software. The Wazuh malware detection capability implements this HIPAA section with the aid of out-of-the-box rules, VirusTotal and YARA integration, and the use of CDB lists. The rootcheck component of Wazuh also detects abnormal behavior in monitored endpoints. These capabilities help support this HIPAA section.
```

Comment:
Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."